pom.xml

```diff
@@ -21,13 +21,6 @@
         <tag>HEAD</tag>
     </scm>
 
-    <developers>
-        <developer>
-            <id>fbelzunc</id>
-            <name>Felix Belzunce Arcos</name>
-        </developer>
-    </developers>
-
     <properties>
         <maven.compiler.source>1.6</maven.compiler.source>
         <maven.compiler.target>1.6</maven.compiler.target>
```

Comment:
Could delete and revert to default of 7, if you updated to the 2.x parent POM, which you should anyway.